python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -644,6 +644,15 @@ def test_schema(self):
     def test_to(self):
         # SPARK-41464: test DataFrame.to()
 
+        cdf = self.connect.read.table(self.tbl_name)
+        df = self.spark.read.table(self.tbl_name)
+
+        def compare_to_schema():
```

Comment:
Can we explicitly pass `df` so we know what is being compared? I would also name it like `assert_eq_schema`

Reply:
`assert_eq_schema` looks is a better name.